routers/web/org/home.go

```diff
@@ -150,8 +152,29 @@ func Home(ctx *context.Context) {
 		return
 	}
 
+	pinnedRepoIDs, err := user_model.GetPinnedRepositoryIDs(org.ID)
+	if err != nil {
+		ctx.ServerError("GetPinnedRepositoryIDs", err)
+		return
+	}
+	var pinnedRepos []*repo_model.Repository
```

Comment:
What is the difference between these? (Go noob!)

Reply:
`var` simply creates a list without prior capacity, with this `make` I specified what capacity to allocate.
This means that for the first `capacity` entries, the slice does not need to be copied to another place once you add something via `append`.
Especially for lists where a lot of items are added, this can make a huge performance difference as then the slice will not need to be copied so often.
Or, as in our case: We know that the slice does not exceed that capacity and hence it will never need to be copied.

Reply:
Ah gotcha!